src/pocketbase/services/record.py

```diff
@@ -30,9 +30,14 @@ async def subscribe(
         self, callback: Callback, record_id: str | None = None, options: CommonOptions | None = None
     ) -> Callable[[], Awaitable[None]]:
         if record_id:
-            return await self._pb.realtime.subscribe(f"{self._collection}/{record_id}", callback, options)
+            # Subscribe to a specific record ID or all records using "*"
+            await self._pb.realtime.subscribe(
+                f"{self._collection}/{record_id}" if record_id != "*" else self._collection,
+                callback, options
+            )
         else:
-            return await self._pb.realtime.subscribe(self._collection, callback, options)
+            # No record ID provided, handle the case explicitly (e.g., raise an error)
+            raise ValueError("Invalid record_id: None")
```

Comment:
I see your motivation for this change is "I modified the subscribe function in record.py to be more faithful to the Pocketbase documentation". I think that is invalid in this case: I do not like the magic "*" character meaning something in a function argument but I do think that making it value `None` is sufficient. I'd ask you to either revert this change _or_ split the subscribe function to a `subscribe` and a `subscribe_all` function or something.